### PR TITLE
DTSAM-572 Audit use of case-allocator rules for CCD case-roles

### PR DESCRIPTION
**Pull request by mikebrownccd**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSAM-572

### Change description ###
DTSAM-572 Audit use of case-allocator rules for CCD case-roles

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/validationrules/core/case-allocator-global.drl [hunk 1]
```diff
@@ -118,4 +132,18 @@ then
     $ra.log("Case Allocator approved : case_allocator_approve_delete_case_role_all_wa_services");
     update($ra);
     logMsg("Rule : case_allocator_approve_delete_case_role_all_wa_services");
+    if (!$ra.getRoleCategory().equals(RoleCategory.LEGAL_OPERATIONS) &&
+        !$ra.getRoleCategory().equals(RoleCategory.JUDICIAL) &&
+        !$ra.getRoleCategory().equals(RoleCategory.ADMIN) &&
+        !$ra.getRoleCategory().equals(RoleCategory.OTHER_GOV_DEPT) &&
+        !$ra.getRoleCategory().equals(RoleCategory.CTSC)) {
+      logInfoMsg("Rule : case_allocator_approve_create_case_role_all_wa_services" +
```
Comment (abul-syed) [inline src/main/resources/validationrules/core/case-allocator-global.drl:140]:
```suggestion
      logInfoMsg("Rule : case_allocator_approve_delete_case_role_all_wa_services" +
```

**Review by abul-syed — CHANGES_REQUESTED** [review]
nice PR, 1 minor suggestion

File: src/main/resources/validationrules/core/case-allocator-global.drl (hunk 1 above)
Comment (mikebrownccd) [reply src/main/resources/validationrules/core/case-allocator-global.drl]: updated

**Review by abul-syed — DISMISSED** [review]
lgtm

**Review by mattnayler — DISMISSED** [review]

**Review by mattnayler — APPROVED** [review]
RE-approved after base branch change.

**Review by abul-syed — APPROVED** [review]
lgtm

**Pull request merged**